### ISSUE TITLE
Searching for headline text in MAP returns 2 results but only 1 is visible

### ISSUE THREAD

**Issue by jroebu14** [bbc/simorgh#5688]
**Describe the bug**
If you go to a MAP url and search for the headline text you will get 2 matches. One for the visible headline and one for the a11y related invisible headline. 

I think this was to accommodate the headline being below the media player and to ensure that a screen reader focuses on the headline first instead of the media player

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://www.bbc.com/pidgin/tori-51645589
2. Search for the headline text text e.g. CMD+F and paste in "https://www.bbc.com/pidgin/tori-51645589"
3. Observe that 2 matches are found but only one is visible.

**Expected behaviour**
There should be only 1 match in the document. This suggests we possibly have a double h1 headline with possible SEO implications and a11y implications

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

**Comment by greenc05** [comment]
There will be 2 matches in the document as this text is on the page twice... There is only 1 x h1. 

"possible SEO implications" can we confirm this is actually an issue for SEO please?

**Comment by jroebu14** [comment]
@greenc05 ah ok I made the assumption the hidden element was also an h1. It appears to be a `strong` element

**Comment by greenc05** [comment]
Lets check with Tom Chandler if there are any SEO implications?

**Comment by aeroplanejane** [comment]
Checking with Tom Chandler - going to move this to icebox until we hear from him

**Comment by aeroplanejane** [comment]
Checked this with Tom Chandler - SEO only shows 1 h1 only, so no impact there - I think this an issue on command F only that I don't believe we can solve given the hidden H1. Seems fairly trivial - would suggest we close.

**Comment by aeroplanejane** [comment]
Closing and removing from the board
